Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/README.md

```diff
@@ -224,7 +279,7 @@ Isolates a machine from accessing external network.
 
 Machine.Isolate
 
-**Note**: When using the Authorization Code Flow, ensure the authenticated user has the “Active Remediation Actions” role assigned. Refer to the [Microsoft documentation on creating and managing roles](https://learn.microsoft.com/en-us/defender-endpoint/user-roles).
+**Note**: When using the Authorization Code Flow (either via the Cortex XSOAR application or by choosing delegated permissions for a self-deployed app), ensure the authenticated user has the “Active Remediation Actions” role assigned. Refer to the [Microsoft documentation on creating and managing roles](https://learn.microsoft.com/en-us/defender-endpoint/user-roles).
```

Comment:
Just to confirm, in [MicrosoftDefenderAdvancedThreatProtection/README.md](https://github.com/demisto/content/pull/40630/files?file-filters%5B%5D=.md&show-viewed-files=true#diff-4ba793b6c30ab0acab6ee410ee2d33176e5d283f3ca8857eaff932e26686c620), and [MicrosoftDefenderAdvancedThreatProtection_description.md](https://github.com/demisto/content/pull/40630/files?file-filters%5B%5D=.md&show-viewed-files=true#diff-9002262d08801af1ea2f62b6ea9f8c07ad3051e36b3a10d45ecdb963197c3079)the term Authentication Code flow is used. Is that different from Authorization Code flow?